TeaDiary.data_access/TeaDiaryContext.cs

```diff
@@ -0,0 +1,21 @@
+﻿using TeaDiary.domain.Models;
+
+namespace TeaDiary.data_access
+{
+    public class TeaDiaryContext : DbContext
+    {
+        public DbSet<Tea> Values { get; set; }
+
+        public TeaDiaryContext()
+            : base("DefaultConnection")
+        {
+
+        }
+
+        protected override void OnModelCreating(DbModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<Tea>().ToTable("Values");
```

Comment:
Why isn't the table named "Teas"?

Reply:
I think I know why. You have extra projects in the file system. You should remove them.

Reply:
Yeah, was readding dataaccess because of wrong project template

Reply:
Deleted excessive one. I hope so at least